src/main/java/org/rulez/demokracia/pdengine/BeatTable.java

```diff
@@ -0,0 +1,48 @@
+package org.rulez.demokracia.pdengine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.persistence.ElementCollection;
+
+import com.sun.tools.javac.util.Pair;
+
+public class BeatTable {
+	@ElementCollection
+	public Map<Choice, HashMap<Choice, Integer>> table;
+
+	public BeatTable() {
+		table = new HashMap<Choice, HashMap<Choice, Integer>>();
+	}
+
+	public int beatInformation(Choice choice1, Choice choice2, String direction) { //change String direction param to Direction
+		//Check 
+		if(direction.equals("DIRECTION_FORWARD"))
```

Comment:
using enum Direction {  FORWARD,  BACKWARD  } instead of strings would be nicer.

Reply:
Yes I know I just created this PR to @magwas , because I am not sure about the data structure so I would not like to start in a wrong way.
Thatswhy I left some comments. :)

Reply:
OK :)